.github/workflows/ci.yml

```diff
@@ -68,21 +68,21 @@ jobs:
           tool: ${{ steps.tool-list.outputs.tool }}
       # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell
       - name: Test bash
-        run: just --version; shfmt --version; protoc --version
+        run: just --version; shfmt --version; protoc --version; osv-scanner --version
```

Comment:
The CI file does not need to be changed because [main.sh checks if the installation is successful](https://github.com/taiki-e/install-action/blob/d140130aeedb5a946a5769684d32e3a33539f226/main.sh#L669).

https://github.com/taiki-e/install-action/actions/runs/7155101623/job/19483349168#step:5:181

The additional checks in these lines are for compatibility with various shells (e.g., windows shells and unix shells have different path formats), and it is sufficient to check each tool for different installation ways one by one.

Reply:
Ok, I'll remove these - they were only added due to the request above.